### DISCUSSION TITLE
casadi 3.6.3 on jetson nano

### DISCUSSION THREAD

**Discussion by fireyan8** [casadi/casadi#3251] [Q&A]
Hallo everyone,
I am using jetson nano, in conda python 3.11, i cannot insall casadi 3.6.3.

The casadi in conda is 3.5.5.   I will get error on 3.5.5
I want to use it for pybamm 23.4.1.   

how to solve this problem or which version of pybamm will require casadi 3.5.5

(pybamm_env) user@ubuntujetson:~/Desktop/MA$ pip install ~/Desktop/MA/casadi-3.6.3-cp310-none-manylinux2014_aarch64.whl
ERROR: casadi-3.6.3-cp310-none-manylinux2014_aarch64.whl is not a supported wheel on this platform.

**Comment by brunoml5** [comment]
Hi!

First, make sure you add the conda-forge channel for your conda installation. Usually it has the last versions for the packages.

Second, it appears you are trying to install the 3.10 version for casadi in your python 3.11 installation (casadi-3.6.3-cp310). Try installing the 3.11 version (casadi-3.6.3-cp311-none-manylinux2014_aarch64.whl).

I hope this solves your problem!
